src/core/locator/locatormodelsuperbridge.h

```diff
@@ -166,6 +166,7 @@ class LocatorModelSuperBridge : public QgsLocatorModelBridge
     void searchRequested( const QString &text );
     void searchTextChangeRequested( const QString &text );
     void locatorFiltersChanged();
+    void requestJumpToPoint( const QgsPoint &center, bool handleMargins = false );
```

Comment:
This is good, but we'll need some more jump requests for activefeature and allfeature locators.